Rakefile

```diff
@@ -51,7 +51,7 @@ Rake::TestTask.new do |t|
   t.libs = ["_docker/lib"]
   t.warning = false
   t.verbose = true
-  t.test_files = FileList['_docker/test/*.rb', '_tests/*.rb'] #Let's add more files here!
+  t.test_files = FileList['_docker/test/**/*.rb', '_tests/*.rb'] #Let's add more files here!
```

Comment:
I've wondered why we named _docker/test instead of _docker/tests but I guess it really doesn't matter.